diff --git a/CosyVoice/cosyvoice/cli/cosyvoice.py b/CosyVoice/cosyvoice/cli/cosyvoice.py
index 3254faa..c464c7a 100644
--- a/CosyVoice/cosyvoice/cli/cosyvoice.py
+++ b/CosyVoice/cosyvoice/cli/cosyvoice.py
@@ -146,7 +146,6 @@ class CosyVoice2(CosyVoice):
             load_jit, load_trt, fp16 = False, False, False
             logging.warning('no cuda device, set load_jit/load_trt/fp16 to False')
 
-        print('device: ', configs['llm'].llm.device, configs['llm'].internlm.device)
         self.model = CosyVoice2Model(configs['llm'].to(device), 
                                      configs['flow'].to(device), 
                                      configs['hift'].to(device), fp16,
